<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>q2d</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,5,0" columnstretch="1,4,0,0,0,0" rowminimumheight="0,0,0" columnminimumwidth="0,0,0,0,0,0">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="projectLayout">
      <item>
       <layout class="QHBoxLayout" name="projectNameLayout">
        <item>
         <widget class="QLabel" name="currentProjectLabel">
          <property name="text">
           <string>Current Project:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="projectNameLabel">
          <property name="text">
           <string>(none)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="newProjectButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create a new empty project and make it the current project&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>new</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="componentTreeLayout" stretch="0,0,0">
      <item>
       <widget class="QLabel" name="componentTreeLabel">
        <property name="text">
         <string>Loaded Components</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="q2d::gui::ComponentTreeView" name="componentTreeView">
        <property name="editTriggers">
         <set>QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addTypeButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>+Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addCategoryButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>+Category</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="documentLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="documentListLabel">
        <property name="text">
         <string>Available Documents</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListView" name="documentListView">
        <property name="editTriggers">
         <set>QAbstractItemView::EditKeyPressed</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="documentDescriptionLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="documentDescriptionText">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="plainText">
         <string>(Not yet implemented)
Edit your description for the current document here</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QTabWidget" name="schematicsTabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="action_createProject"/>
    <addaction name="action_loadProject"/>
    <addaction name="action_saveProject"/>
    <addaction name="action_closeProject"/>
    <addaction name="separator"/>
    <addaction name="action_createDocument"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProject"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_createProject">
   <property name="text">
    <string>Create Project</string>
   </property>
   <property name="toolTip">
    <string>Create a new empty project and make it the current project</string>
   </property>
  </action>
  <action name="action_loadProject">
   <property name="text">
    <string>Load Project</string>
   </property>
  </action>
  <action name="action_saveProject">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="action_closeProject">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="action_createDocument">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create Document</string>
   </property>
   <property name="toolTip">
    <string>Create a new empty document in the current project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>q2d::gui::ComponentTreeView</class>
   <extends>QTreeView</extends>
   <header>gui/ComponentTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
